src/shared/project-manager-review/ProjectManagerReview.tsx

```diff
@@ -0,0 +1,102 @@
+import { i18n } from '@lingui/core';
+import {
+  Box,
+  ExpansionPanel,
+  ExpansionPanelDetails,
+  ExpansionPanelSummary,
+  makeStyles,
+  Theme,
+  Typography,
+} from '@material-ui/core';
+import { CSSProperties } from '@material-ui/core/styles/withStyles';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import graphql from 'babel-plugin-relay/macro';
+import React from 'react';
+import { useFragment } from 'react-relay/hooks';
+import { Evaluation } from 'src/global-types';
+import { EvaluationOutput } from 'src/shared/evaluationt-output';
+import { MultilineOutput } from 'src/shared/multiline-output';
+import { FCProps } from 'src/shared/types/FCProps';
+import { Styles } from 'src/shared/types/Styles';
+import { ProjectManagerReview_review$key } from './__generated__/ProjectManagerReview_review.graphql';
+
+interface OwnProps {
+  review: ProjectManagerReview_review$key;
+}
+
+type Props = FCProps<OwnProps> & StyleProps;
+
+export function ProjectManagerReview(props: Props) {
+  const review = useFragment(fragment, props.review);
+  const classes = useStyles(props);
+
+  return (
+    <ExpansionPanel
+      classes={{ root: classes.expansionPanelRoot, expanded: classes.expansionPanelExpanded }}
+      defaultExpanded={true}
+      elevation={0}
+    >
+      <ExpansionPanelSummary expandIcon={<ExpandMoreIcon />}>
+        <Typography variant="h6">{review.project.name}</Typography>
+      </ExpansionPanelSummary>
+      <ExpansionPanelDetails>
+        <div className={classes.wrapper}>
+          <div>
+            <Typography gutterBottom>{i18n._('Performance to initial expectation')}: </Typography>
+            <Box className={classes.wrapperBox}>
+              {review.rating ? <EvaluationOutput value={review.rating as Evaluation} /> : <p></p>}
```

Comment:
I think null is better
```suggestion
              {review.rating ? <EvaluationOutput value={review.rating as Evaluation} /> : null}
```

Reply:
Will use typography variant body 1 here with non breaking space